exchange/events_test.go

```diff
@@ -192,3 +195,351 @@ func Test_isEventAllowed(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertToVastEvent(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    config.Events
+		expected injector.VASTEvents
+	}{
+		{
+			name: "Error event",
```

Comment:
Nitpick: `t.Run` does not allow test names to have spaces and will auto fill them in. To make it easier to associate a test failure, please do not use spaces in test names. Consider formats like `error-event`.